### PR TITLE
Avalon Labs: new markets for DeFi lending

### PR DESCRIPTION
**Pull request by jasmin-avalon**
Avalon DeFi Lending new markets:
- Kaia Main Market: https://app.avalonfinance.xyz/?marketName=proto_kaia_v3
- Zetachain Main Market: https://app.avalonfinance.xyz/?marketName=proto_zetachain_v3
- BSC ListaDAO Market: https://app.avalonfinance.xyz/?marketName=proto_bsc_lista_innovation
- Taiko USDa Market

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/avalon-finance-usdalend exports TVL: 
        
 
 
    iotex                     1.49 M
    taiko                     0
    taiko-borrowed            0
    iotex-borrowed            0
    borrowed                  0
    
    total                    1.49 M

**Comment by llamatester** [comment]
The adapter at projects/avalon-finance-usdalend exports TVL: 
        
 
 
    iotex                     1.49 M
    iotex-borrowed            0
    borrowed                  0
    
    total                    1.49 M

**Comment by llamatester** [comment]
The adapter at projects/avalon-finance exports TVL: 
        
 
 
    btr                       261.90 M
    core                      260.97 M
    borrowed                  225.17 M
    bsc                       129.93 M
    taiko                     106.65 M
    merlin                    90.10 M
    bsc-borrowed              77.57 M
    bob                       77.21 M
    btr-borrowed              69.57 M
    core-borrowed             68.84 M
    merlin-borrowed           5.85 M
    arbitrum                  5.28 M
    bob-borrowed              2.29 M
    ethereum                  2.06 M
    iotex                     1.70 M
    arbitrum-borrowed         967.15 k
    base                      249.04 k
    base-borrowed             56.08 k
    ethereum-borrowed         28.90 k
    scroll                    10.54 k
    iotex-borrowed            2.48 k
    bsquared                  1.32 k
    zeta                      447
    bsquared-borrowed         303
    mode                      216
    klaytn                    34
    zeta-borrowed             32
    taiko-borrowed            0
    scroll-borrowed           0
    mode-borrowed             0
    klaytn-borrowed           0
    
    total                    936.06 M

File: projects/avalon-finance/index.js [hunk 1]
```diff
@@ -68,7 +70,15 @@ const lbtcMarkets = {
 	bsc: aaveExports('', '', undefined, ['0x5b9b3C211B81627Cc6b46824CB26829F31A587dc'], { v3: true }),
 }
 
+const listaMarkets = {
+	bsc: aaveExports('', '', undefined, ['0x5157f63bE7808DEB090Eee7762e917745896A09E'], { v3: true }), // BSC - ListaDAO
+}
+
+const usdaDefiMarkets = {
```
Comment (g1nt0ki) [inline projects/avalon-finance/index.js:77]: wait, this is doublecounting, removing this

**Pull request merged**